addons/addon-base-raas/packages/base-raas-workflow-steps/lib/steps/provision-account/provision-account.js

```diff
@@ -205,11 +251,22 @@ class ProvisionAccount extends StepBase {
           name,
           roleArn: cfnOutputs.CrossAccountExecutionRoleArn,
           xAccEnvMgmtRoleArn: cfnOutputs.CrossAccountEnvMgmtRoleArn,
-          subnetId: cfnOutputs.VpcPublicSubnet1,
           vpcId: cfnOutputs.VPC,
           encryptionKeyArn: cfnOutputs.EncryptionKeyArn,
         };
-        await this.addAwsAccountTable(requestContext, awsAccountData);
+        let additionalAccountData = {};
+        if (this.settings.get(settingKeys.isAppStreamEnabled) === 'true') {
+          additionalAccountData = {
```

Comment:
Should we add other properties as well like fleet or workspace subnet?

Reply:
@SanketD92 correct me if I'm wrong, but the only additional fields that you need to get the connect API working is `appStreamStackName` and `appStreamSecurityGroup`?

Reply:
That's correct. The stack name is enough to get the associated AppStream resources.

However if there are more than 1 fleets associated to that AppStream stack (edge-case), that would result in an error while connecting. So if we're adding fields anyways I'd vote to add `appStreamFleetName` as well.

Workspace subnet is not needed for connect API changes, so I'll leave that up to you.

Reply:
Added `appStreanFleetName`. Workspace subnet is needed so SWB knows which subnet to launch workspaces in.

Reply:
Yeah, that's correct. We'll need to add subnet too.